extensions/interactions/GraphInput/directives/graph-detail.service.ts

```diff
@@ -33,6 +33,32 @@ export class GraphDetailService {
   VERTEX_RADIUS: number = 6;
   EDGE_WIDTH: number = 3;
 
+  // The minimum margin to the left and top of the display.
+  // The first vertex should be at (MIN_MARGIN, MIN_MARGIN).
+  // This value should not be 0 because only half of the vertex will be shown.
+  MIN_MARGIN: number = 10;
```

Comment:
Can you make this MIN_MARGIN_PX (or whatever units it is)?

Reply:
Done